### ISSUE TITLE
[Accessibility][Functional]: 'Basic uncontrolled example (allowFreeform: T, AutoComplete: T)' combo box saves blank values.

### ISSUE THREAD

**Issue by joschect** [OfficeDev/office-ui-fabric-react#11196]
Split out of issue #6920

**Comment by xugao** [comment]
i think it's a very confusing UX to allow saving blank value. m not sure why we decided to do this. shouldn't saving blank value means to not select any options? @jspurlin could you give more context here?

**Comment by jspurlin** [comment]
You can look back through bugs that existed. That functionality was added due to a request from a consumer

**Comment by xugao** [comment]
ok putting a closure on this issue:
1. The control by design allows saving blank values
2. regarding "screen reader should be conveying some appropriate information to the users when focus goes on blank value", i disagree. if you look at control such as `TextField`, it's behaving the same. I don't think user needs extra information here.

Going to close this issue as by design
